server/app/Http/Middleware/CorsHeader.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+
+class CorsHeader
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        $response = $next($request);
+        $response->header("Access-Control-Allow-Origin" , "*");
+        $response->header("Access-Control-Allow-Methods" , "GET, POST, PUT, DELETE, OPTIONS");
+//        dd("asds");
```

Comment:
do not leave this kind of comments when you commit

Reply:
This middleware should be deleted. It doesn't work anyway.